### ISSUE TITLE
Hotkeys

### ISSUE THREAD

**Issue by Madis0** [Aizistral-Studios/No-Chat-Reports#381]
### Idea

NCR could have a set of hotkeys for people to bind. None of those should be set to a key by default to prevent accidents/confusion.

### Reasoning

Accessibility and convenience

### Other Information

- [ ] Chat state toggle
- [ ] Global config

Toggles should show an actionbar text indicating current value when pressed.

**Comment by Aizistral** [comment]
I like all of these except global config, it feels like that would be better done with a GUI button. In principle we could extend the functionality of config reload button in multiplayer menu to allow access to actual config screen with, say, right-click, and simple reload with left-click. Obviously the screen still depends on Cloth Config API to work, so it won't be accessible in any way without it.

**Comment by Aizistral** [comment]
> I like all of these except global config, it feels like that would be better done with a GUI button.

I think that is exactly what #349 suggests, just with a safety status button instead of the one in multiplayer menu. Not sure which one would work better.

**Comment by Madis0** [comment]
I agree that #349 should be implemented and I agree that it should be implemented in the status icon, as that is contextual. 
Implementing it in the refresh icon seems unnecessary as the user can 
- use Mod Menu
- edit the config file (which is what the refresh button is for) or 
- even have an additional button for config alone as the screen space is not limited, nor do buttons obstruct anything in the server screen

**Comment by Madis0** [comment]
Either way implementing a hotkey to config as well would be similar to minimap mods for example, where people may want to edit their config often. I personally don't think I'd need it either, it was just suggested to "complete the set".

**Comment by Secret-chest** [comment]
Please disable them by default!

**Comment by IlyaTheSawmill** [comment]
> Please disable them by default!

Why?

It is best to set it in the vanilla key settings.
